Update about-me.md

markgurarie opened the pull request:
Uploaded the bio from your friend; made some copy edits, blended some material from the original with what the friend sent. Yeah?

markgurarie committed:
Update about-me.md

Uploaded the bio from your friend; made some copy edits, blended some material from the original with what the friend sent. Yeah?

zgypa commented on about-me.md:
```diff
@@ -4,31 +4,12 @@ layout: duckone
-
-Toni is always interested in collaborating with others, since his projects are
-often too big for him to complete alone. Reach out to
-[email@antoniomagni.com](mailto:email@antoniomagni.com) to connect!
```
Is there any reasone why you removed the reach out connect stuff?

markgurarie replied:
added back!

zgypa commented on about-me.md:
```diff
@@ -4,31 +4,12 @@ layout: duckone
 style: members
 title: About Toni
 ---
+Toni Magni is a biomedical engineer focused on orthodontic software IT infrastructures. Currently employed at tops Software (Atlanta, GA), he serves as co-chair for the ADA Standards Committee for Dental Informatics Working Group 11.6 (Integration of Orthodontic Standards) and Clinical Instructor in the Department of Orthodontics at Case Western Reserve University. A frequent collaborator with va ...
```
...and _is_ Clinical instructor... ?

zgypa replied:
I'm not sure the last sentence actually captures properly what I'm doing. The more extensive text that was here before might've done a better job. I'm open to discussing this with you may be in a phone call, so that you can understand what I would like to express and help me put it with the right words.

markgurarie committed:
Update about-me.md

OK, reverse order: added back email- duh- you're all about collaboration....
Took a listen to what you said, and made some adjustments: thought it would be good to emphasize more the open-source/open to collaboration/ground-up changes thing. Thought it good also to include you thinking about policy/"attacking" the whole system, as it were. Does that look better? More?  Let me know.  
Oh and thought "mindfulness = success" is a bit of a platitude (many factors go into success-seems a bit simple, no?), so I tucked that back a bit to you believing in it in the workplace, and then that leading you to your practice and interest in meditation. But you know you can reject that change.

markgurarie committed:
Update about-me.md

zgypa commented on about-me.md:
```diff
@@ -4,31 +4,8 @@ layout: duckone
 style: members
 title: About Toni
 ---
+Toni Magni is a biomedical engineer focused on orthodontic software IT infrastructures. Currently employed at tops Software (Atlanta, GA), he serves as co-chair of the ADA Standards Committee for Dental Informatics Working Group 11.6 (Integration of Orthodontic Standards) and Clinical Instructor in the Department of Orthodontics at Case Western Reserve University. He is fuelled by a passion for e ...
```
...unnecessarily difficult...?

zgypa replied:
While true that i am mainly focussed on dicom imaging for orthodontics (not only installing servers) right now, that's not the only thing i do to promote interoperability as i perceive it here. Does it actually sound like I perceived it? 

The Python and JAVA libs are ongoing. I perceive as if i wrote them and now i'm done. Its more like i develop and maintain on GitHub open source Python and JAVA code/tools to save and share...

zgypa replied:
While true that i am mainly focussed on dicom imaging for orthodontics (not only installing servers) right now, that's not the only thing i do to promote interoperability as i perceive it here. Does it actually sound like I perceived it?

zgypa replied:
The Python and JAVA libs are ongoing. I perceive as if i wrote them and now i'm done. Its more like i develop and maintain on GitHub open source Python and JAVA code/tools to save and share...

zgypa commented:
For collaboration purposes, how about hard wrapping the lines, or splitting sentences one per line? Would be easier to comment, since i think i can only comment a single line at a time.

markgurarie commented:
Hard wrapping the lines. I assume that means adding some sort of code, which I can look up. Unfortunately, I'm a bit under the gun right now, but I can take a look at that tomorrow (maybe late tonight? sort of doubt it) and figure it out so that it is easier for collaboration yep.
